### PR TITLE
chore: delete outdated text

### PR DESCRIPTION
**Pull request by omarsilva1**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: docs/pages/deployment/update.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ as the latest release. This will look as follows, e.g.:
 ```json
 {
   "title": "Dataspace Connector",
-  "description": "IDS Connector originally developed by the Fraunhofer ISST",
+  "description": "IDS Connector",
   "version": "6.2.0",
   "contact": {
-    "organization": "Fraunhofer Institute for Software and Systems Engineering",
+    "organization": "sovity",
     "website": "https://www.dataspace-connector.io/",
```
Comment (SebastianOpriel) [inline docs/pages/deployment/update.md]: Remove

File: docs/pages/deployment/update.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ as the latest release. This will look as follows, e.g.:
...
     "email": "support@sovity.de"
```
Comment (SebastianOpriel) [inline docs/pages/deployment/update.md]: Contact@

File: docs/pages/deployment/update.md [hunk 3]
```diff
@@ -29,10 +29,10 @@ as the latest release. This will look as follows, e.g.:
 ```json
 {
   "title": "Dataspace Connector",
-  "description": "IDS Connector originally developed by the Fraunhofer ISST",
+  "description": "IDS Connector",
   "version": "6.2.0",
   "contact": {
-    "organization": "Fraunhofer Institute for Software and Systems Engineering",
+    "organization": "sovity",
```
Comment (SebastianOpriel) [inline docs/pages/deployment/update.md]: sovity GmbH

**Review by SebastianOpriel — CHANGES_REQUESTED** [review]

File: docs/pages/deployment/update.md (hunk 3 above)
Comment (omarsilva1) [reply docs/pages/deployment/update.md]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**